committee-steering/governance/sig-governance.md

```diff
@@ -1,60 +1,136 @@
 # SIG Roles and Organizational Governance
 
-This charter adheres to the conventions described in the [Kubernetes Charter README]. It will be updated as needed to meet the current needs of the Kubernetes project.
+This charter adheres to the conventions described in the [Kubernetes Charter README].
+It will be updated as needed to meet the current needs of the Kubernetes project.
 
-In order to standardize Special Interest Group efforts, create maximum transparency, and route contributors to the appropriate SIG, SIGs should follow these guidelines:
+In order to standardize Special Interest Group efforts, create maximum
+transparency, and route contributors to the appropriate SIG, SIGs should follow
```

Comment:
Should we simply link any of the required items that are found here and under the `### Chair` section rather duplicate in both places?

Reply:
im 100% on board with that. 

message for the audience: spoke to lachie in DMs on slack. we are going to merge and iterate on this section. before completely removing / reorganizing this part, i need to make sure the details are covered below, too. .